### DISCUSSION TITLE
Have a kind of Safe Exam Browser for tests/exams

### DISCUSSION THREAD

**Discussion by marcothijs** [dodona-edu/feedback-and-support#55] [Feature requests]
With ChatGPT available any Python solution can be found in seconds during tests and exams. Would be good to a have a kind of Safe Exam Browser like in Bookwidgets that prevent students to access other applications or sites.

**Comment by jonathanvynck** [comment]
Yes please!

**Comment by bmesuere** [comment]
I'm afraid there is not much we can do here as a website 😞

We'll keep our eyes and ears open for solutions, but there's nothing planned in the near future.

**Reply by marcothijs** [reply]
Thanks for your reply. 
I'm using Bookwidgets for tests at school and that is also a website (online platform). They use Safe Exam Browser, an extra piece of software that needs to be installed by the pupils so that they (when switched this mode on for a test) can't access other sites.

**Reply by pdawyndt** [reply]
Safe Exam Browser does not seem to be compatible with Linux (only Windows, Mac OS and iOS).

**Reply by marcothijs** [reply]
Ok, didn't know that. Thanks for explaining.

**Comment by jonathanvynck** [comment]
A colleague of mine has used dodona with SEB. I'll ask him for the details.

**Reply by pdawyndt** [reply]
We're definitely open for a demo as well (we could organize a video call).

**Reply by jonathanvynck** [reply]
Seems like he just set up SEB so students can go to one of his courses in which he hid all exercises except the ones used for the exam.

**Comment by jonathanvynck** [comment]
I would really like an automatic SEB-integration in Dodona like Bookwidgets has.
It's a pain to set everything up manually for different exams, students, ...
